templates/mail/auth/reset_passwd.tmpl

```diff
@@ -7,7 +7,9 @@
 
 {{ $recover_url := printf "%suser/recover_account?code=%s" AppUrl .Code}}
 <body>
-	{{.i18n.Tr "mail.reset_password.body" .DisplayName .ResetPwdCodeLives $recover_url | Str2html}}
+	<p>{{.i18n.Tr "mail.hi_user_x" .DisplayName | Str2html}}</p><br>
+	<p>{{.i18n.Tr "mail.reset_password.text" .ResetPwdCodeLives | Str2html}}</p><p><a href="{{$recover_url}}">{{$recover_url}}</a></p><br>
```

Comment:
But is converted to HTML here - will it escape the `<b>` tags, too?

Reply:
yes Str2html is here for the `<b>abc</b>`

Reply:
Ah, got it. Sorry, I thought it would escape the text, because "Str2html" sounded like it would convert `<b>` to `&lt;b&gt;`or something, but I now got what it's about markdown rendering. I find the name a little unintuitive, as you can see.